h_program-lang/Parse_info.ml

```diff
@@ -534,8 +534,9 @@ let complete_token_location_large filename table x =
  *    skip them easily with one Common.exclude
  *)
 let tokenize_all_and_adjust_pos file tokenizer visitor_tok is_eof =
- Common.with_open_infile file (fun chan -> 
-  let lexbuf = Lexing.from_channel chan in
+ Common.with_open_infile file (fun chan ->
+  let string = Unicode.UTF8.asciify chan in
```

Comment:
maybe we can do this only if the file contains a unicode char in the first place?

Reply:
I was just worried of slowing down a bit things only because of a few files. but maybe it is negligible.